stdlib/2and3/cgitb.pyi

```diff
@@ -13,21 +13,21 @@ else:
     _Path = Union[_T]
 
 
-def reset() -> str: ...
-def small(text: str) -> str: ...
-def strong(text: str) -> str: ...
-def grey(text: str) -> str: ...
-def lookup(name: str, frame: FrameType, locals: Dict[str, Any]) -> Tuple[Optional[str], Any]: ...
-def scanvars(reader: Callable[[], bytes], frame: FrameType, locals: Dict[str, Any]) -> List[Tuple[str, Optional[str], Any]]: ...
+def reset() -> str: ...  # undocumented
+def small(text: str) -> str: ...  # undocumented
+def strong(text: str) -> str: ...  # undocumented
+def grey(text: str) -> str: ...  # undocumented
+def lookup(name: str, frame: FrameType, locals: Dict[str, Any]) -> Tuple[Optional[str], Any]: ...  # undocumented
+def scanvars(reader: Callable[[], bytes], frame: FrameType, locals: Dict[str, Any]) -> List[Tuple[str, Optional[str], Any]]: ...  # undocumented
 def html(einfo: _ExcInfo, context: int = ...) -> str: ...
 def text(einfo: _ExcInfo, context: int = ...) -> str: ...
 
-class Hook:
+class Hook:  # undocumented
 
     def __init__(self, display: int = ..., logdir: Optional[_Path[AnyStr]] = ..., context: int = ..., file: Optional[IO[str]] = ..., format: str = ...) -> None: ...
     def __call__(self, etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
     def handle(self, info: Optional[_ExcInfo] = ...) -> None: ...
 
-handler: Callable[[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]], None]
+def handler(etype: Optional[Type[BaseException]], evalue: Optional[BaseException], etb: Optional[TracebackType]) -> None: ...
```

Comment:
This should just have one `info` argument, though (like `Hook.handle()`).

Reply:
Oh whoops, copied from the wrong source function (And I had just looked at the documentation, too)